src/main/java/dog/pawbook/model/managedentity/owner/Owner.java

```diff
@@ -99,6 +102,14 @@ public String toString() {
         return builder.toString();
     }
 
+    public void addDogId(int dogId) {
+        this.dogIdList.add(dogId);
+    }
+
+    public ArrayList<Integer> printDogIdList() {
+        return this.dogIdList;
+    }
```

Comment:
there shouldn't be a need to do this as well.

Reply:
Modified it.